nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Add tag

# Description
Tag management at full CRUD. All tag operations occur in the TagManagement route. No additional URL routes
## Type of change


# Testing Instructions
User may view all available tags in Tag Management. Clicking on a listed tag will open the edit window which allows editing and deleting of the tag. Deleting requires a confirm which opens in place of the edit window. Clicking Add New will open the Add fields in the same position as the edit window. The Edit window is on the right in large format mode, above the tag list in mobile format mode.
# Checklist:
